coalib/settings/ConfigurationGathering.py

```diff
@@ -201,12 +210,13 @@ def warn_config_absent(sections, argument, log_printer=None):
         argument = [argument]
     for section in sections.values():
         if any(arg in section for arg in argument):
-            return False
+            return (False, argument)
 
-    formatted_args = ' or '.join('`--{}`'.format(arg) for arg in argument)
-    logging.warning('coala will not run any analysis. Did you forget '
-                    'to give the {} argument?'.format(formatted_args))
-    return True
+    if not (only_global_bears and argument == ['files']):
+        formatted_args = ' or '.join('`--{}`'.format(arg) for arg in argument)
+        logging.warning('coala will not run any analysis. Did you forget '
+                        'to give the {} argument?'.format(formatted_args))
```

Comment:
Will coala still run analysis, even though this message doesn't appear if only global bears given?

Reply:
does it matter? as no files are provided. is there a difference between coala running analysis on no files and coala running no analysis at all? @Makman2